examples/glfw/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.15)
+project(FlutterEmbedderGLFW)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11" )
+
+add_executable(flutter_glfw FlutterEmbedderGLFW.cc)
+
+############################################################
+# GLFW
+############################################################
+find_path(GLFW_INCLUDE_PATH "glfw3.h" /usr/local/Cellar/glfw/3.3/include/GLFW/)
```

Comment:
Please use `find_package` to use pkg-config so this is cross platform.

Reply:
In order for that to work I think you need something like this: https://github.com/Kitware/CMake/blob/master/Modules/FindFLTK.cmake . They don't have one for GLFW.

Also brew doesn't install a .pc file for GLFW so I don't think pkg-config can work for the mac build.

I added extra paths that should catch standard installs on linux.  I couldn't test it right now because unfortunately I'm locked out of my linux box.  People can always edit the variables in ccmake.